WordPress/gutenberg
# Latest Comments block: minor code cleanup

## Description
This PR reorders the attributes in the Latest Comments block's `index.php` alphabetically and removes an unnecessary `''` item from the `enum` array of the `align` object that has no effect on what classes get applied and what alignments are considered valid.

Notably, the values in `enum` actually do have an effect on what alignments are considered valid. There is no documentation for this, as far as I can tell, and no way to use this same `enum` array in JavaScript.

## How to test
Try setting alignments on the Latest Comments block. Ensure that the correct classes (or lack thereof) are applied in both the editor and the front-end. If you notice an issue, compare to `master` as it may be a bug there as well.
